packages/xo-server/src/api/vif.js

```diff
@@ -174,6 +174,6 @@ set.params = {
 }
 
 set.resolve = {
-  vif: ['id', 'VIF', 'operate'],
+  vif: ['id', 'VIF', 'administrate'],
```

Comment:
A VIF should be editable by a VIF operator. However, for settings that have an impact on the network, [the user should also have admin ACLs on the network](https://github.com/vatesfr/xen-orchestra/issues/4700#issuecomment-576348136).